src/components/InstallTabs/index.tsx

```diff
@@ -15,6 +15,7 @@ const InstallTabs = (): JSX.Element => {
     MAC: ['nvm (macOS)', 'Chocolatey (Windows)', 'nvm (Linux)'],
     LINUX: ['nvm (Linux)', 'nvm (macOS)', 'Chocolatey (Windows)'],
     UNKNOWN: ['Chocolatey (Windows)', 'nvm (macOS)', 'nvm (Linux)'],
+    // MOBILE: ['Chocolatey (Windows)', 'nvm (macOS)', 'nvm (Linux)'],
```

Comment:
Delete?

Reply:
hah, yeah. removed. changed the logic a bit since we're using css to hide this anyway on mobile